language-plutus-core/src/Language/PlutusCore/Renamer.hs

```diff
@@ -83,39 +81,39 @@ runScopedRenameM = runRenameM emptyScopedUniquesRenaming
 
 -- | The empty 'UniquesRenaming'.
 emptyUniquesRenaming :: UniquesRenaming unique
-emptyUniquesRenaming = UniquesRenaming mempty
+emptyUniquesRenaming = mempty
```

Comment:
I'd probably inline this now

Reply:
I inlined it, then returned it back, to make `runDirectRenameM` and `runScopedRenameM` look symmetric.

Reply:
If you take my suggestion below they will both be `mempty`.